packages/chain-bridge/src/helpers/BpsUtils.sol

```diff
@@ -5,6 +5,9 @@ import {Percentage, PERCENTAGE_FACTOR} from "@summerfi/percentage-solidity/contr
 import {PercentageUtils} from "@summerfi/percentage-solidity/contracts/PercentageUtils.sol";
 import {Bps, BPS_FACTOR, toBps, fromBps} from "./Bps.sol";
 
+uint256 constant BPS_PER_PERCENT = BPS_FACTOR / 100; // 1% == 100 bps
+uint256 constant PERCENTAGE_PER_BPS = PERCENTAGE_FACTOR / BPS_PER_PERCENT; // 1 bps == 1e16
```

Comment:
Bad constants, in Solidity you should always multiply and then divide to minimize rounding errors